src/Hl7.Fhir.Core/Hl7.Fhir.Core.csproj

```diff
@@ -17,9 +18,16 @@
     <RootNamespace>Hl7.Fhir</RootNamespace>
   </PropertyGroup>
 
+   <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net45|AnyCPU'">
+     <DefineConstants>TRACE;USE_CODE_GEN</DefineConstants>
+   </PropertyGroup>
+
   <ItemGroup>
```

Comment:
This ItemGroup can be removed: it was not there in R3 as well

Reply:
Done